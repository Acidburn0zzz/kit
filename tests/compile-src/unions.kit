include "stdio.h";

union MyUnion {
    public static var xyz = "hi";

    var a: Int;
    var b: CString;
}

function main() {
    var a: MyUnion;
    a.a = 100;
    printf("%li\n", a.a);
    a.b = "hello world";
    printf("%s\n", a.b);
}
