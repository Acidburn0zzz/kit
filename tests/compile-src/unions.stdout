hi
100
abcdefg
